templates/components/entry_cross.html.twig

```diff
@@ -7,7 +7,7 @@
         data-action="notifications:Notification@window->subject#notification">
     <aside class="meta entry__meta">
         {% if entry.visibility in ['visible', 'private'] or (entry.visibility is same as 'trashed' and this.canSeeTrashed) %}
-            <a href="{{ entry_url(entry) }}">{{ 'crossposted'|trans }}</a> {{ 'by'|trans }}
+            {{ 'Related'|trans }}: {% if entry.isAdult %}<small class="danger">18+</small>{% endif %} <a href="{{ entry_url(entry) }}">{{ entry.title }}</a> {{ 'by'|trans }}
```

Comment:
I think maybe I should try using the twig filter `truncate` on the entry title to a smaller length, seems simple enough and might keep it from going crazy number of lines on mobile

might also want padding though, it looks a bit more busy on mobile

Reply:
we don't have the twig extension for truncate it turns out, and thinking about it I don't _actually_ want to chop the text off, overflow hidden/ellipsis is superior as it would be better for screenreaders